zerver/tests/test_users.py

```diff
@@ -830,7 +830,7 @@ def test_create_user_with_multiple_streams(self) -> None:
 
         prereg_user = PreregistrationUser.objects.get(email="fred@zulip.com")
 
-        with self.assert_database_query_count(93):
+        with self.assert_database_query_count(94):
```

Comment:
This increase in query is also because of the reason mentioned in https://github.com/zulip/zulip/pull/27544/commits/b2b3594ac11e3ff549a96ab76b7ceeabd759497c#r1382436152.